src/camvidlog2/vid.py

```
@@ -11,7 +11,13 @@
 
 
 class FrameError(Exception):
-    """Exception raised when there's an issue related to video frames."""
+    """Exception raised when related to video frames."""
+
+    pass
```

Ruff (0.11.9)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)